packages/ppx_deriving/ppx_deriving.4.3/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "whitequark <whitequark@whitequark.org>"
+authors: [ "whitequark <whitequark@whitequark.org>" ]
+license: "MIT"
+homepage: "https://github.com/ocaml-ppx/ppx_deriving"
+doc: "https://whitequark.github.io/ppx_deriving"
```

Comment:
Do you need to change the maintainer and bugs address too? Since it is under ocaml-ppx now.